### PR TITLE
Removed duplicate code in SkeletonCollectionDelegate

### PR DESCRIPTION
**Pull request by keshavamurthy1**
Removed check for older versions in UIView+Frame
Removed return on single lines (Implicit Returns)

### Summary

Removed duplicate code and older version check

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/Juanpe/SkeletonView/blob/develop/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://github.com/Juanpe/SkeletonView/blob/develop/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: Sources/Extensions/UIView+Frame.swift [hunk 1]
```diff
@@ -37,11 +37,7 @@ extension UIView {
     }
     
     var isRTL: Bool {
-        if #available(iOS 10.0, *), #available(tvOS 10.0, *) {
```
Comment (Juanpe) [inline Sources/Extensions/UIView+Frame.swift:40]: **`SkeletonView`** is compatible with iOS 9, so we must leave this check here.

**Comment by Juanpe** [comment]
Hi @keshavamurthy1, thanks for this change, I left a comment about a minor change :)

**Comment by keshavamurthy1** [comment]
@Juanpe Updated PR with changes for the review comment

File: Sources/Extensions/UIView+Frame.swift (hunk 1 above)
Comment (Juanpe) [reply Sources/Extensions/UIView+Frame.swift]: 👍🏼

**Comment by Juanpe** [comment]
> @Juanpe Updated PR with changes for the review comment

Thanks!

**Pull request merged**